.github/workflows/acceptance-tests.yml

```diff
@@ -302,16 +302,17 @@ jobs:
           scoop install twilio
           twilio --version
           twilio api --help
-      - name: Draft - Perform testing on scoop installations
-        env:
-          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
-        run: |
-          (New-Object System.Net.WebClient).DownloadFile('https://get.scoop.sh', 'scoop.ps1') 
-          ./scoop.ps1 -RunAsAdmin
-          scoop bucket add twilio-scoop https://github.com/twilio/scoop-twilio-cli
-          scoop install twiliodraft
-          twilio --version
-          twilio api --help
+      # enable on draft branch or with an isDraft flag
+      # - name: Draft - Perform testing on scoop installations
+      #   env:
+      #     GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+      #   run: |
+      #     (New-Object System.Net.WebClient).DownloadFile('https://get.scoop.sh', 'scoop.ps1') 
+      #     ./scoop.ps1 -RunAsAdmin
+      #     scoop bucket add twilio-scoop https://github.com/twilio/scoop-twilio-cli
+      #     scoop install twiliodraft
+      #     twilio --version
+      #     twilio api --help
```

Comment:
Do we know the reason for these failures?

Reply:
Not exactly, the times I [ran](https://github.com/shrutiburman/twilio-cli/runs/5810830564?check_suite_focus=true) it on forked repo, it worked perfectly. Could have been an intermittent issue. However, this block is commented as it's not applicable for a main branch release.